tests/integration/suite/test_tokens.py

```diff
@@ -84,7 +84,7 @@ def test_kubeconfig_token_ttl(admin_mc, user_mc):
     assert token1 != token2
 
     # reset kubeconfig ttl setting
-    client.update_by_id_setting(id="kubeconfig-token-ttl-minutes",
+    client.update_by_id_setting(id="kubeconfig-default-token-ttl-minutes",
                                 value="960")
```

Comment:
The previous default value was 960 but is that still the case with the new setting? Seems like it's default value is "0".